File: Java/PrimeNumber.java
// PrimeNumber.java
public class PrimeNumber {
    public static boolean isPrime(int num) {
        if (num <= 1) return false;
        if (num <= 3) return true;
        if (num % 2 == 0 || num % 3 == 0) return false;
        
        for (int i = 5; i * i <= num; i += 6) {
            if (num % i == 0 || num % (i + 2) == 0) {
                return false;
            }
        }
        return true;
    }

    public static void main(String[] args) {
        int testNum = 11;
        System.out.println("Is " + testNum + " a prime number? " + isPrime(testNum));
        
        int testNum2 = 25;
        System.out.println("Is " + testNum2 + " a prime number? " + isPrime(testNum2));
    }
}

File: Python/prime_number.py
# prime_number.py

def is_prime(num):
    """Check if a number is prime."""
    if num <= 1:
        return False
    for i in range(2, int(num**0.5) + 1):
        if num % i == 0:
            return False
    return True

if __name__ == "__main__":
    test_num = 11
    result = is_prime(test_num)
    print(f"{test_num} is {'prime' if result else 'not prime'}")
